.github/workflows/build.yml

```diff
@@ -116,6 +116,7 @@ jobs:
     runs-on: windows-latest
     needs: check_source
     if: needs.check_source.outputs.run_tests == 'true'
+    continue-on-error: true
```

Comment:
By default, all the jobs listed in the `check` job's `needs` section are required to succeed in terms of branch protection. This setting emulates having the job non-required (in combination with the check job below being the only thing required in branch protection).

Reply:
This has been moved to the action call. Resolving.